hammett/widgets/base.py

```diff
@@ -52,7 +52,13 @@ async def _post_render(
         config: 'FinalRenderConfig',
         **kwargs: 'Any',
     ) -> None:
-        """Save to user_data initialized state after screen rendering if it's new message."""
+        """Save to user_data initialized state after screen rendering if it's new message.
+
+        Raises
+        ------
+            MissingPersistence: If widgets are used in jobs with no specified persistence.
```

Comment:
`MissingPersistence: If the widgets are used in jobs with no specified persistence.`